locales-pending/limitations-apply-premium.ftl

```diff
@@ -2,8 +2,22 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-rules-and-restrictions-pill = Rules & Restrictions
 # Variables:
-# $discount_percentage_num is the amount discounted in percentage per month.
-# $discount_duration is the number of month(s) that users will pay the discounted price.
-limitations-apply-header = { $discount_percentage_num } off the next { $discount_duration } months
+# $discount_percentage_num is the amount discounted in percentage per month
+# $discount_duration is the number of month(s) that users will pay the discounted price
+settings-limitations-apply-title =
+    { $discount_duration ->
+        [one] { $discount_percentage_num } off { $discount_duration } month promotion
+       *[other] { $discount_percentage_num } off { $discount_duration } months promotion
+}
+settings-limitations-pill = Rules & Restrictions
+settings-limitations-apply-description-one = Discount available for a limited time only.
+settings-limitations-apply-description-two = Redeemable 1 time only.
+# Variables:
+# $discount_percentage_num is the amount discounted in percentage per month
+# $discount_duration is the number of month(s) that users will pay the discounted price
+settings-limitations-apply-description-three =
```

Comment:
This might make more sense, given that it's not on the settings page?

```suggestion
limitations-apply-title =
    { $discount_duration ->
        [one] { $discount_percentage_num } off { $discount_duration } month promotion
       *[other] { $discount_percentage_num } off { $discount_duration } months promotion
}
limitations-pill = Rules & Restrictions
limitations-apply-description-one = Discount available for a limited time only.
limitations-apply-description-two = Redeemable 1 time only.
# Variables:
# $discount_percentage_num is the amount discounted in percentage per month
# $discount_duration is the number of month(s) that users will pay the discounted price
limitations-apply-description-three =
```